extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/runtime/LiquibaseMongodbConfig.java

```diff
@@ -15,6 +16,12 @@
 @ConfigRoot(name = "liquibase-mongodb", phase = ConfigPhase.RUN_TIME)
 public class LiquibaseMongodbConfig {
 
+    /**
+     * The name of mongo client
+     */
+    @ConfigItem(defaultValue = MongoClientBeanUtil.DEFAULT_MONGOCLIENT_NAME)
+    public String mongoClientName;
```

Comment:
Do we want to be able to target a specific client or should we support migrations of potentially all the MongoDB clients?

Reply:
Basically we should have the ability to decide which mongo client to use for running liquibase (The proposed solution here).

An extent to this would be implementing named liquibase configurations where you can create multiple liquibase configurations each operates on specific mongo client as needed (Same as you can create named mongo clients to connect agains different databases). This would allow us to run multiple migration scripts against different mongo clients through configurations.

I was exploring the codebase of mongoclient to see how named client were implemented, but I don't feel confident enough to propose same concept for liquibase within this PR before fully understanding how build steps work in quarkus.